### ISSUE TITLE
[0.4.0] TypeError: Cannot use 'in' operator to search for 'subscribe' in ASC

### ISSUE THREAD

**Issue by danielg2002** [Urigo/graphql-modules#303]
works in 0.2.18 but fails on 0.3.0+

schema:

enum ConnectionOrderDirection {
    ASC
    DESC
  }

TypeError: Cannot use 'in' operator to search for 'subscribe' in ASC
    at _loop_2 (/Users/daniel/Code/OrangeLoops/phoenix-server/node_modules/@graphql-modules/core/dist/graphql-module.js:446:42)
    at _loop_1 (/Users/daniel/Code/OrangeLoops/phoenix-server/node_modules/@graphql-modules/core/dist/graphql-module.js:477:21)
    at GraphQLModule.addSessionInjectorToSelfResolversContext (/Users/daniel/Code/OrangeLoops/phoenix-server/node_modules/@graphql-modules/core/dist/graphql-module.js:483:13)
    at GraphQLModule.buildSchemaAndInjector (/Users/daniel/Code/OrangeLoops/phoenix-server/node_modules/@graphql-modules/core/dist/graphql-module.js:659:30)
    at GraphQLModule.get [as injector] (/Users/daniel/Code/OrangeLoops/phoenix-server/node_modules/@graphql-modules/core/dist/graphql-module.js:133:22)
    at GraphQLModule.buildSchemaAndInjector (/Users/daniel/Code/OrangeLoops/phoenix-server/node_modules/@graphql-modules/core/dist/graphql-module.js:592:43)
    at GraphQLModule.get [as schema] (/Users/daniel/Code/OrangeLoops/phoenix-server/node_modules/@graphql-modules/core/dist/graphql-module.js:117:22)

**Comment by danielg2002** [comment]
Also I think you should update the typescript definition file at least for GraphQLModule's "imports" option... I'm using an array of strings with the name of modules.

**Comment by ardatan** [comment]
@danielg2002 After 0.4.0, we changed the way for importing modules. So, you have to use module itself for `imports` option.
For the bug related to enum, you can use the following canary alpha versions to make it work;
```
Successfully published:
 - @graphql-modules/core@0.5.0-alpha.7cb9422c
 - @graphql-modules/di@0.5.0-alpha.7cb9422c
 - @graphql-modules/epoxy@0.5.0-alpha.7cb9422c
 - @graphql-modules/sonar@0.5.0-alpha.7cb9422c
 - @graphql-modules/utils@0.5.0-alpha.7cb9422c
```

**Comment by danielg2002** [comment]
@ardatan thank you! now works!

**Comment by ardatan** [comment]
Available since 0.4.1
